apps/mobile/src/screens/profile/profile.screen.tsx

```diff
@@ -1,19 +1,37 @@
-import { Button } from 'native-base';
+import { Ionicons } from '@expo/vector-icons';
+import { useNavigation } from '@react-navigation/native';
+import { Box, Button, FlatList, Icon, Text } from 'native-base';
 
 import Delimiter from '@/components/delimiter/delimiter';
 import SafeArea from '@/components/safe-area/safe-area';
 import Topbar from '@/components/topbar/topbar';
 import { useAuth } from '@/hooks/use-auth-provider';
 import useLocale from '@/hooks/use-locale';
+import Routes from '@/routes';
 
 const ProfileScreen = () => {
   const { signOut } = useAuth();
   const { t } = useLocale();
 
+  const { navigate } = useNavigation();
+
   return (
     <SafeArea>
       <Delimiter flex="1">
         <Topbar />
+        <FlatList
+          data={ROUTES}
+          keyExtractor={({ name }) => `app-route-${name}`}
+          renderItem={({ item }) => (
+            <Button
+              onPress={() => navigate(item.path as never)}
+              variant="ghost"
+              leftIcon={<Icon as={Ionicons} name="paw" size="lg" color="primary.300" />}
+            >
+              <Text color="primary.300">{t(item.translateKey)}</Text>
+            </Button>
```

Comment:
I think a `Pressable` component is more convenient to be used here. The text should be also left aligned